.github/labeler.yml

```diff
@@ -0,0 +1,11 @@
+Needs tech review:
+- changed-files:
+  - any-glob-to-any-file: ['**']
+
+Needs tech review:
+- changed-files:
+  - any-glob-to-any-file: ['**']
+
+Needs testing:
+- changed-files:
+  - any-glob-to-any-file: ['guides/**']
```

Comment:
FYI: This will label the PR if there are changes in any file in the `guides` folder or its subfolders.